src/components/components/MobileNavigation/MobileNavigation.scss

```diff
@@ -0,0 +1,40 @@
+.MobileNavigation__wrapper {
+    display: none;
+}
+
+@media screen and (max-width: 480px) {
```

Comment:
please remove the entire media query

Reply:
let's discuss then our approach here, please? @jasper-prgh

Reply:
I guess Jasper wants us to do something like this instead in the App.js:
`{window.innerWidth > 480 ? <Navbar /> : <MobileNavigation />}`

Reply:
That'd ve been my appoach, yes. But it's your choiuce. I will keep myself out of this. Sorry!

Reply:
I like it